brian2/devices/device.py

```diff
@@ -412,10 +414,10 @@ def spike_queue(self, source_start, source_end):
         # Use the C++ version of the SpikeQueue when available
         try:
             from brian2.synapses.cythonspikequeue import SpikeQueue
-            logger.info('Using the C++ SpikeQueue', once=True)
+            logger.diagnostic('Using the C++ SpikeQueue', once=True)
         except ImportError:
             from brian2.synapses.spikequeue import SpikeQueue
-            logger.info('Using the Python SpikeQueue', once=True)
+            logger.diagnostic('Using the Python SpikeQueue', once=True)
```

Comment:
Something like this good be a good idea (maybe just collect information about this in a documentation page and refer to that in the warning?) -- but I'd say let's open a new (post 2.0) issue about that. Somewhat related, I think it would be good to have a function that allows something like `print brian_configuration()` and would give you the codegen target, the spike queue, the version of Brian and its dependencies, etc. (similar to what we have in the log), for quick inspection.